Src/IronPython/Runtime/Operations/ListOfTOps.cs

```diff
@@ -53,5 +55,70 @@ public static class ListOfTOps<T> {
                 infinite.RemoveAt(index);
             }
         }
+
+        #region Python __ methods
+
+        public static bool __contains__(List<T> l, T item) {
```

Comment:
I think they're generated by some of the `MemberResolver`s defined in `PythonTypeInfo`